brush/package-lock.json

```diff
@@ -12230,6 +12230,12 @@
       "version": "file:../scissors",
       "requires": {
         "ramda": "0.26.1"
+      },
+      "dependencies": {
+        "ramda": {
+          "version": "0.26.1",
+          "bundled": true
+        }
```

Comment:
Since you doesn't change anything at `dependencies` from `package.json`, you should not commit changes at `package-lock.json`.

Please, do a `git rebase` to remove this changes at the commit